configs/DAKEFPVF722_HD/config.h

```diff
@@ -0,0 +1,126 @@
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        DAKEFPVF722_HD
+#define MANUFACTURER_ID   DAKE
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+#ifndef USE_CAMERA_CONTROL
+#define USE_CAMERA_CONTROL
+#endif
+
+#define BEEPER_PIN           PC3
+#define LED_STRIP_PIN        PA0
+#define MOTOR1_PIN           PA1
+#define MOTOR2_PIN           PA2
+#define MOTOR3_PIN           PA3
+#define MOTOR4_PIN           PA8
+#define MOTOR5_PIN           PA9
+#define MOTOR6_PIN           PA10
+#define MOTOR7_PIN           PC8
+#define MOTOR8_PIN           PC9
+#define UART1_TX_PIN         PB6
+#define UART4_TX_PIN         PC10
+#define UART5_TX_PIN         PC12
+#define UART6_TX_PIN         PC6
+#define UART1_RX_PIN         PB7
+#define UART3_RX_PIN         PB11
+#define UART4_RX_PIN         PC11
+#define UART5_RX_PIN         PD2
+#define UART6_RX_PIN         PC7
+#define I2C1_SCL_PIN         PB8
+#define I2C1_SDA_PIN         PB9
+#define LED0_PIN             PA15
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI3_SCK_PIN         PB3
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PB14
+#define SPI3_SDI_PIN         PB4
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PB15
+#define SPI3_SDO_PIN         PB5
+#define CAMERA_CONTROL_PIN   PB0
+#define ADC_VBAT_PIN         PC1
+#define ADC_RSSI_PIN         PC2
+#define ADC_CURR_PIN         PC0
+#define PINIO1_PIN           PB1
+#define PINIO2_PIN           PB10
+#define FLASH_CS_PIN         PB2
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_1_CS_PIN        PA4
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PA1 , 1, 0) \
+    TIMER_PIN_MAP( 1, PA2 , 1, 0) \
+    TIMER_PIN_MAP( 2, PA3 , 1, 0) \
+    TIMER_PIN_MAP( 3, PA8 , 1, 1) \
+    TIMER_PIN_MAP( 4, PA9 , 1, 1) \
+    TIMER_PIN_MAP( 5, PA10, 1, 1) \
+    TIMER_PIN_MAP( 6, PC8 , 2, 1) \
+    TIMER_PIN_MAP( 7, PC9 , 2, 0) \
+    TIMER_PIN_MAP( 8, PA0 , 2, 0) \
+    TIMER_PIN_MAP( 9, PB0 , 2, -1)
+
+#define ADC2_DMA_OPT        1
+
+#define ADC_INSTANCE ADC2
+#define FLASH_SPI_INSTANCE SPI3
+#define MAX7456_SPI_INSTANCE SPI2
+
+#define BARO_I2C_INSTANCE I2CDEV_1
+#define MAG_I2C_INSTANCE I2CDEV_1
+
+#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_FLASH
+#define DEFAULT_DSHOT_BITBANG DSHOT_BITBANG_ON
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+
+#define BEEPER_INVERTED
+#define PINIO1_CONFIG 129
+#define PINIO1_BOX 40
+#define PINIO2_CONFIG 129
+#define PINIO2_BOX 41
```

Comment:
You probably want BT to be switched off when armed, so set the PINIO2 box to 0. And possibly change PINIO2 config to 1 depending on how the switch is wired. 
```suggestion
#define PINIO2_BOX 0
```